extensions/duplicate_message_policing.py

```diff
@@ -46,9 +46,12 @@ async def delete_duplicate(event: hikari.GuildMessageCreateEvent) -> None:
         or event.is_webhook
         or event.is_bot
         or event.content.startswith(nodelete_flag)
-        or "http" in event.content  # allow links
-        or "@" in event.content  # allow mentions
         or len(event.content) <= 2  # allow short messages
+        or "http" in event.content  # allow links
+        or re.match("<@\\d+>", event.content)
+        or re.fullmatch(
+            "<:[a-z]+:\\d+>", event.content
```

Comment:
Animated emoji have an `a` before the first `:`.

Reply:
![image](https://github.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/blob/master/Emojis/Animals/Cat%20Face.png?raw=true)

actioned